swarm/network/kademlia.go

```diff
@@ -536,71 +570,127 @@ func (k *Kademlia) Prune(c <-chan time.Time) {
 	}()
 }
 
+// PeerPot keeps info about expected nearest neighbours and empty bins
+// used for testing only
+type PeerPot struct {
```

Comment:
would it be an idea to move testing code like this to separate files and/or have a naming scheme for them (prefix or suffix Tmp for example) so it's easier to weed them out later, and avoid that people start using them if they linger?